storage/src/nodestore/header.rs

```diff
@@ -224,22 +225,22 @@ impl NodeStoreHeader {
     }
 
     /// Get the free lists
-    pub const fn free_lists(&self) -> &FreeLists {
+    pub fn free_lists(&self) -> &FreeLists {
```

Comment:
Why were these removed? They didn't seem to cause any problems when I replaced them.